### PR TITLE
New version: Hashicorp.Nomad version 1.7.4

### PR DESCRIPTION
**Pull request by weimeng**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138823)

### CODE REVIEW

**Comment by weimeng** [comment]
What do I do here? I'm asked to conform to the 1.6 schema in the PR template, which specifies `ManifestType: locale`, but the checks are asking that I conform to the 1.4 schema, which specifies `ManifestType: defaultLocale`.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-12T09:28:34.4410923Z ##[error] Manifest Error: The multi file manifest is incomplete.A multi file manifest must contain at least version, installer and defaultLocale manifest.

(Automated response - build 720.)

File: manifests/h/Hashicorp/Nomad/1.7.4/Hashicorp.Nomad.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Copyright: Copyright (c) 2023 HashiCorp, Inc.
+CopyrightUrl: https://github.com/hashicorp/nomad/blob/main/LICENSE
+ShortDescription: Nomad is an easy-to-use, flexible, and performant workload orchestrator that can deploy a mix of microservice, batch, containerized, and non-containerized applications.
+Description: Nomad is an easy-to-use, flexible, and performant workload orchestrator that can deploy a mix of microservice, batch, containerized, and non-containerized applications. Nomad is easy to operate and scale and has native Consul and Vault integrations.
+Documentations:
+- DocumentLabel: Nomad Documentation
+  DocumentUrl: https://developer.hashicorp.com/nomad/docs
+- DocumentLabel: Tutorials
+  DocumentUrl: https://developer.hashicorp.com/nomad/tutorials
+ManifestType: locale
```
Comment (stephengillie) [inline manifests/h/Hashicorp/Nomad/1.7.4/Hashicorp.Nomad.locale.en-US.yaml:25]:
```suggestion
ManifestType: defaultLocale
```

**Comment by stephengillie** [comment]
> What do I do here? I'm asked to conform to the 1.6 schema in the PR template, which specifies `ManifestType: locale`, but the checks are asking that I conform to the 1.4 schema, which specifies `ManifestType: defaultLocale`.

Hi @weiming,

The [1.6.0 Default Locale manifest](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/defaultLocale.md) page uses `ManifestType: defaultLocale` for the default localization page for the software package. `ManifestType: locale` is used for [additional localization files](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/locale.md).

**Comment by weimeng** [comment]
Thanks, I see. The filename confused me.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 788.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**